Source/ASEditableTextNode.mm

```diff
@@ -842,4 +842,30 @@ - (void)_delegateDidFinishEditing
     [_delegate editableTextNodeDidFinishEditing:self];
 }
 
+
+- (BOOL)isAccessibilityElement
```

Comment:
We are treating `ASEditableTextNode` as a `UIAccessibilityContainer` now, so this method should return `NO`. You should also be able to just remove it as by default it will return `NO`. Some more details on `UIAccessibilityContainer`: https://developer.apple.com/documentation/uikit/accessibility/uiaccessibilitycontainer

Reply:
Good catch! Was a carry over from when I had it working previously without #1188. Removed now.